app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::UsersController < Api::BaseController
+
+  def update
+    @user = User.find(params[:id])
```

Comment:
Also you don't need the parentheses here, you can just do `User.find params[:id]`.